src/pages/SectionPage.jsx

```diff
@@ -60,6 +68,15 @@ const sectionsData = [
   },
 ];
 
+function CreateNewSectionSegment(onAddSegment, addSection) {
+  const { isOpen, onOpen, onClose } = useDisclosure();
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'onClose' is assigned a value but never used.